besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1342,8 +1342,7 @@ private void parse(
     // and eventually it will run regular parsing of the remaining options.
 
     final ConfigOptionSearchAndRunHandler configParsingHandler =
-        new ConfigOptionSearchAndRunHandler(
-            resultHandler, exceptionHandler, CONFIG_FILE_OPTION_NAME, environment);
+        new ConfigOptionSearchAndRunHandler(resultHandler, exceptionHandler, environment);
 
```

Comment:
@macfarla got rid of injecting the config option name, felt like it just added confusion.

Reply:
👍